udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -179,6 +183,10 @@ <h2 class="subtitle">{{ _('Informations') }}</h2>
                                  <dd>
                                      {{dataset.id}}
                                  </dd>
+                                {% if external_source %}
+                                <dt class="f-bold text-grey-400">{{ _('Remote source') }}</dt>
```

Comment:
Why this styling? I think we can keep the `dt` style only?

Reply:
My bad, it came from the old sources